client/src/popups/DeleteContent.tsx

```diff
@@ -98,23 +47,13 @@ export function DeleteContent({
     >
       <ModalOverlay />
       <ModalContent>
-        <ModalHeader textAlign="center">
-          {errMsg ? (
-            <>Error moving to trash</>
```

Comment:
So what happens now if there is an error when moving to the trash? Or is it just silly to check for errors here since we don't check anywhere else?